javascript/ql/lib/semmle/javascript/frameworks/NodeJSLib.qll

```diff
@@ -423,6 +425,47 @@ module NodeJSLib {
     |
       n = "filename" or n.regexpMatch("(old|new|src|dst|)path")
     )
+    or
+    fsExtraExtensionFileParam(methodName, i)
+  }
+
+  /**
+   * Holds if `methodName` is a function defined in the `fs-extra` library
+   * as an extension to node.js' `fs` module and parameter `i` of of the
+   * method might represent a file path.
+   */
+  private predicate fsExtraExtensionFileParam(string methodName, int i) {
```

Comment:
@asgerf Could I get your opinion here? Should we skip directly to models-as-data for this predicate?

More generally: should we have (rough) guidelines for when we prefer models-as-data over QL implementations?

Reply:
👍  good question!

Reply:
There's generally no pressure to use MaD at the moment.

Reply:
Alright, that settles it then, @esbena?